vector/src/main/res/layout/view_url_preview.xml

```diff
@@ -7,83 +7,79 @@
     android:layout_height="wrap_content"
     tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout">
 
-    <View
-        android:id="@+id/url_preview_left_border"
-        android:layout_width="2dp"
-        android:layout_height="0dp"
-        android:background="?vctr_content_tertiary"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <com.google.android.material.card.MaterialCardView
```

Comment:
Can the class `PreviewUrlView` inherits from MaterialCardView instead, to remove a hierarchy level here? (this is maybe not possible)

Reply:
good idea! will check...

Reply:
Seems to work here

Reply:
https://github.com/vector-im/element-android/pull/4610/commits/41884f83d84f02d850ee626a24c4747d064171b1 also includes the formatting (assuming the formatter settings have been picked up)